README.md

```diff
@@ -0,0 +1,35 @@
+# ElasticSearch on Rancher
+
```

Comment:
Its good practice to have a short summary of the repo at the top. See https://github.com/RichardLitt/standard-readme/blob/master/spec.md for more info.

If you don't want to read that, just add a sentence or two prepended with an `>` so the text is rendered as quoted.